packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/interfaces/NotificationsReconstructor.java

```diff
@@ -0,0 +1,11 @@
+package expo.modules.notifications.notifications.interfaces;
+
+import android.os.Parcel;
+
+import expo.modules.notifications.notifications.model.NotificationRequest;
+
+public interface NotificationsReconstructor {
```

Comment:
```suggestion
@FunctionalInterface
public interface NotificationsReconstructor {
```

Reply:
This one, however, is not intended to be Functional. It just so happens it has one method. But I guess we should use this annotation only when we really want it to stay this way, while I have nothing against expanding this interface in the future, according to our needs.